app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -479,6 +479,15 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         return browsers.isDefaultBrowser
     }
 
+    var shownDefaultBrowserNotification by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_should_show_default_browser_notification),
+        default = false
+    )
+
+    fun shouldShowDefaultBrowserNotification(): Boolean {
```

Comment:
nit: Maybe move this to where ever it's only use case is. It doesn't seem like this function needs to be globally accessible from all of settings. It's a strange pattern that seems to exist in this class for some reason. 🤷

Reply:
Sounds good.  I'll move it thanks,

Reply:
Hmm, this makes testing more complicated.  I think I'll leave this where it is.  I would like to be able to test this without other dependencies.